projects/storefrontlib/cms-components/content/paragraph/paragraph.component.html

```diff
@@ -1 +1,4 @@
-<p *ngIf="component.data$ | async as data" [innerHTML]="data.content"></p>
+<p
+  *ngIf="component.data$ | async as data"
+  [innerHTML]="data.content | cxAnchor"
```

Comment:
The word 'anchor' means a _link_ in general.
The goal of our pipe is very specific - to prepend the current URL to links containing only a hash fragment in the `href` attribute (or so called [URI/URL fragment](https://en.wikipedia.org/wiki/URI_fragment)).

IMHO It would be good to have more precise name. What do you think about the following?
`[innerHTML]="data.content | cxSuplementHashAnchors`

Reply:
Decided on 
```
[innerHTML]="data.content | cxSuplementHashAnchors export class SupplementHashAnchorsPipe {}
```